test_scripts/API/VehicleData/HandsOffSteering/024_OnVD_notification_disallowed.lua

```diff
@@ -0,0 +1,57 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0257-New-vehicle-data-HandsOffSteering.md
+--
+-- Description: Check that SDL doesn't transfer OnVehicleData notification to App if an app not allowed by policy with
+-- new 'handsOffSteering' parameter
+--
+-- Preconditions:
+-- 1) Update preloaded_pt file, add handsOffSteering parameter to VD_RPC group
+-- 2) OnVehicleData notification is Not allowed by policies
+-- 3) App is registered
+-- 4) App is subscribed on handsOffSteering parameter
+-- Steps:
+-- 1) HMI sends valid VehicleInfo.OnVehicleData notification to SDL
+-- SDL does:
+-- - a) ignored this notification and not transfer to App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/HandsOffSteering/common')
+local json = require("modules/json")
+
+--[[ Local Variables ]]
+local value = { true, false }
+local rpc_sub = "SubscribeVehicleData"
+local VDGroup = {
+  rpcs = {
+    SubscribeVehicleData = {
+      hmi_levels = { "NONE", "BACKGROUND", "LIMITED", "FULL" },
+      parameters = {"handsOffSteering"}
+    },
+    OnVehicleData = {
+      hmi_levels = { "NONE", "BACKGROUND", "LIMITED", "FULL" },
+      parameters = json.EMPTY_ARRAY
+    }
+  }
+}
+
+--[[ Local Function ]]
+local function onVDNotAllowed(pHandsOffSteering)
```

Comment:
Redefine function for OnVD in common file to process cases with receiving notification and not